src/translib/transformer/sw_portchannel.go

```diff
@@ -167,6 +179,35 @@ var YangToDb_lag_min_links_xfmr FieldXfmrYangToDb = func(inParams XfmrParams) (m
     return res_map, nil
 }
 
+var DbToYang_lag_min_links_xfmr FieldXfmrDbtoYang = func(inParams XfmrParams) (map[string]interface{}, error) {
+    if log.V(3) {
+        log.Info("Entering DbToYang_lag_min_links_xfmr")
+    }
+    var err error
+    result := make(map[string]interface{})
+
+    intfType, _, ierr := getIntfTypeByName(inParams.key)
+    if ierr != nil || intfType != IntfTypePortChannel  {
+        log.Error("Invalid Interface")
+        return result, err
+    }
+    data := (*inParams.dbDataMap)[inParams.curDb]
+    links, ok := data[PORTCHANNEL_TABLE][inParams.key].Field["min_links"]
+    if ok {
```

Comment:
Can't we use the existing get_min_links() API here ?

Reply:
get_min_links() again makes a call to DB to get min-links, (*inParams.dbDataMap) will be more efficient. get min-links value I have reduced some repetitive code by adding new function  uint16Conv